BFWControls Demo/Modules/Table/Controller/ExcludeCellDemoViewController.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ExcludeCellDemoViewController.swift
+//  BFWControls
+//
+//  Created by Andy Kim on 9/6/17.
+//  Copyright © 2017 BareFeetWare. All rights reserved.
+//
+
+import UIKit
+import BFWControls
+
+class ExcludeCellDemoViewController: StaticTableViewController {
+
+    var selectedCells =  [UITableViewCell]()
+    
+    override var excludedCells: [UITableViewCell] {
+        return selectedCells
+    }
+    
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        removeCell(indexPath: indexPath)
+    }
+
+    fileprivate func removeCell(indexPath: IndexPath) {
+        if let cell = tableView.cellForRow(at: indexPath) {
+            tableView.beginUpdates()
+            selectedCells.append(cell)
+            tableView.deleteRows(at: [indexPath], with: .automatic)
+            tableView.endUpdates()
```

Comment:
Is `beginUpdates()` and `endUpdates()` needed here? Doesn't the single `deleteRows()` do its own animation?